### ISSUE TITLE
Training it on VCTK Dataset

### ISSUE THREAD

**Issue by pbansal5** [NVIDIA/tacotron2#378]
I have been facing some issues with training of a multispeaker Tacotron2 on VCTK dataset. For this I have append the LSA vector (512 dimension) with 256 dim speaker embeddings. The same setup worked quite well when trained with Google Speech Commands (which has some noisy samples and unaligned utterances).

How should I modify my model to account for the same? All the hyperparameters are the same as the standard ones for the LJ Speech dataset

The reduced_losse I am getting are around 0.3

I am attaching the image for the training curves for the same here : 

<img width="1424" alt="Screenshot 2020-06-19 at 7 10 35 PM" src="https://user-images.githubusercontent.com/34788639/85138540-9125ce80-b260-11ea-969e-3f08fdcd0f49.png">

**Comment by rafaelvalle** [comment]
What are the issues you're facing and what is LSA?
Please zoom in into your validation loss such that it does not look like a straight line. Say form 2k onwards

**Comment by pbansal5** [comment]
By LSA I mean the Location Sensitive Attention (the output of the encoder). In one of issues you had recommended appending speaker embedding to LSA to make multispeaker work. I did see reasonable results in Speech commands. 
The problem is that even the training examples are not being reproduced (training loss seems stagnant). So I didn't zoom in on the validation loss (which too is stagnant btw).
I was looking for a MultiSpeaker Generative model for a recent project and hence tried to train one. I digressed into some other repos too (like mellotron and flowtron)
I don't know if this is the right platform but I was wondering when would the Flowtron pretrained models comparable to tacotron would be uploaded. I tried to infer from it, but the outputs there too were not comparable to tacotron.

**Comment by rafaelvalle** [comment]
Take a look at how the speaker embedding is incorporated to the model in [Mellotron](https://github.com/NVIDIA/mellotron/)

What training regime are you in?

If you're underfitting, you can decrease [weight decay](https://github.com/NVIDIA/tacotron2/blob/master/hparams.py#L82), increase the model size, etc, things that will help the model achieve lower losses.

If you're overfitting and you'r training loss is going down while you're validation loss is going up, then you'll need to augment your data or decrease model capacity.

**Comment by pbansal5** [comment]
Sure. Also would you recommend training it on VCTK as compared to LibriTTS?

**Comment by rafaelvalle** [comment]
Take a look at the properties of each dataset and evaluate which fits your needs.
